fe/fe-core/src/main/java/com/starrocks/catalog/SchemaTable.java

```diff
@@ -725,6 +725,17 @@ public static Builder builder() {
                                     .column("IDLE", ScalarType.createType(PrimitiveType.BOOLEAN))
                                     .column("FINISHED_TASKS", ScalarType.createType(PrimitiveType.BIGINT))
                                     .build()))
+                    .put("be_logs", new SchemaTable(
+                            SystemId.BE_THREADS_ID,
+                            "be_logs",
+                            TableType.SCHEMA,
+                            builder()
+                                    .column("BE_ID", ScalarType.createType(PrimitiveType.BIGINT))
+                                    .column("LEVEL", ScalarType.createVarchar(NAME_CHAR_LEN))
+                                    .column("TIMESTAMP", ScalarType.createType(PrimitiveType.BIGINT))
+                                    .column("TID", ScalarType.createType(PrimitiveType.BIGINT))
+                                    .column("LOG", ScalarType.createVarchar(NAME_CHAR_LEN))
```

Comment:
```suggestion
                                    .column("MESSAGE", ScalarType.createVarchar(NAME_CHAR_LEN))
```
Do you think `message` or `log_message` is a better name than `log`?

Reply:
I see, the `LOG` column contains the full content of the log, not only the log message.

Reply:
I don't have any preference, it's just a name :)